.github/workflows/tests-integration.yaml

```diff
@@ -71,8 +71,6 @@ jobs:
 
       - name: run integration tests
         timeout-minutes: 10
-        env:
-          HIVE_DEBUG: 1
```

Comment:
This causes stack traces to be rendered by OClif which breaks our CLI snapshot tests.